cli/lib/commands/init.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+
+module.exports = create;
+
+const readline = require('readline');
+const {
+  createPackageJson,
+  createToml,
+  fileExists,
+  isValidSemver,
+  writeFile
+} = require('../utils');
+
+const REJECTION_MSGS = {
+  REQUIRED: 'This is required',
+  validateName: "Sorry, that's not a valid name",
+  validateVersion: "Sorry, that's not a valid semver version"
+};
+
+const VALID_YES_NO = {
+  y: 1,
+  yes: 1,
+  n: 0,
+  no: 0
+};
+
+const QUESTIONS = {
+  TOML_EXISTS: {
+    MESSAGE: 'Package.toml already exists. Continue (Y/N)?: ',
+    REQUIRED: true
+  },
+  NAME: {
+    MESSAGE: 'Name: ',
+    REQUIRED: true
+  },
+  VERSION: {
+    MESSAGE: 'Version: ',
+    REQUIRED: true
+  },
+  ENTRY: {
+    MESSAGE: 'Entry: ',
+    REQUIRED: false
+  },
+  TYPE: {
+    MESSAGE: 'Type :',
```

Comment:
I've noticed an inconsistency here - sometimes you would have space before `:` and sometimes you don't

Reply:
updated